Sweeper/Views/PinActions/PinActionsView.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  PinActionsView.swift
+//  Sweeper
+//
+//  Created by Wuming Xie on 10/20/17.
+//  Copyright © 2017 team11. All rights reserved.
+//
+
+import UIKit
+
+protocol PinActionsViewDelegate: class {
+    func pinActions(_ pinActionsView: PinActionsView, didLike: Bool)
+    func pinActions(_ pinActionsView: PinActionsView, didComment: Bool)
```

Comment:
I think if you're not passing anything back (didLike and didComment should always be true when these are called), you should name delegates as: `pinActionsViewDidLike(_ pinActionsView: PinActionsView)` and `pinActionsViewDidComment(...)`

Reply:
Will update this.